### PR TITLE
Add aarch64 and ppc64le architectures

### PR DESCRIPTION
**Pull request by ehfd**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ehfd** [comment]
@conda-forge-admin, please rerender

**Comment by ehfd** [comment]
@conda-forge-admin, please rerender

**Comment by ehfd** [comment]
@conda-forge-admin, please rerender

**Comment by ehfd** [comment]
@conda-forge-admin, please rerender

**Comment by ehfd** [comment]
@hmaarrfk The main intent is to make my currently private (later public) stack build without workarounds I don't like, but this actually has legitimate applications; VA-API is an alternative pathway to video4linux encoding/decoding for aarch64 embedded devices.

**Comment by ehfd** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -32,9 +33,9 @@ requirements:
     - xorg-libxext
     - xorg-libxfixes
   run:
-    - xorg-libx11
```
Comment (hmaarrfk) [inline recipe/meta.yaml:35]: this seems to be required. or at least `libxcb`

**Comment by hmaarrfk** [comment]
> VA-API is an alternative pathway to video4linux encoding/decoding for aarch64 embedded devices.

Honestly, i thought VA-API was just for intel processors.

Happy to have help on the video stack!

**Comment by hmaarrfk** [comment]
I somewhat just built this library without thinking too much about the implications, I wanted to use the Intel hardware encoder (it didn't give videos that were of high quality enough, and was only available on a certain subset of processors so it wasn't too fun).

I'm not sure if X11 is required for the core functionality, or just some demo application.

**Comment by ehfd** [comment]
https://github.com/selkies-project/selkies-gstreamer

This is my project and this is why I'm so keen on making GStreamer and X11 work in Conda. It's because I want low-latency (better than) VNC in HPC environments.
Do check it out when you have time. @hmaarrfk We'll probably have a release in ~ 48 hours and this briefly stopped that progress.

The X11 library is still required as a pathway to support devices as an alternative to DRM and if there was a `wayland-scanner` feedstock it would also build on Wayland.

```
$ vainfo --help
Show information from VA-API driver
Usage: vainfo --help
        --help print this message

Usage: vainfo [options]
  -a, --all                              Show all supported attributes
Display options:
        --display display | help         Show information for the specified display, or the available display list 
        --device device                  Set device name, only available under drm display
$ vainfo --display help
Available displays:
  wayland
  x11
  drm
```

It can also expand to Windows (https://devblogs.microsoft.com/directx/video-acceleration-api-va-api-now-available-on-windows/).

And it's also the only way in Linux for now to encode with AMD until Vulkan + AMF works in GStreamer.

File: recipe/meta.yaml (hunk 1 above)
Comment (ehfd) [reply recipe/meta.yaml:35]: It's because I added fail when overlinked in conda-build. Added libxcb as well as dependencies (rightfully).

Comment (ehfd) [reply recipe/meta.yaml:35]: Build succeeds. @hmaarrfk

**Comment by hmaarrfk** [comment]
we have some starting seeds of wayland. Excited to see this get through!

**Comment by ehfd** [comment]
@hmaarrfk Any reason the CI isn't running?

**Pull request merged**